### PR TITLE
nspawn: allow tinting the background of interactive tty sessions

### PR DESCRIPTION
**Pull request by poettering**
this just hooks up the tinting logic we added for the uid0 tool also for nspawn

(background: i noticed that the new gnome terminals all do similar coloring of the header bar, so i figured we can do the same with the background)

### CODE REVIEW

File: src/shared/pretty-print.c [hunk 1]
```diff
@@ -441,3 +442,30 @@ int conf_files_cat(const char *root, const char *name, CatFlags flags) {
 
         return cat_files(path, files, flags);
 }
+
+int terminal_tint_color(double hue, char **ret) {
+        double red, green, blue;
+        int r;
+
```
Comment (bluca) [inline src/shared/pretty-print.c:449]: assert(ret)

**Review by bluca — APPROVED** [review]
Please add bash completion for --background - just for the parameter, can't meaningfully complete the value of course

**Comment by poettering** [comment]
let's please not make bash completion a requirement for PRs. there's no testing logic for it. I can add stuff there, but it will be copypasta mess, and noone will notice.

**Comment by poettering** [comment]
I mean, looking into the nspawn shell completion I am pretty sure it's not at all right. for example --no-new-privileges appears to accept args according to the shell completion, but it does not.

**Comment by bluca** [comment]
> let's please not make bash completion a requirement for PRs. there's no testing logic for it. I can add stuff there, but it will be copypasta mess, and noone will notice.

I know there's no testing and it's less than ideal, but shell completion is very important for usability. But it can be added later, doesn't matter

**Pull request merged**